common-docs/extensions/approval.md

```diff
@@ -5,4 +5,16 @@ GitHub Organizations or GitHub repos can be approved.
 
 Banned extensions are not allowed to load.
 
-## #approval
\ No newline at end of file
+## Checklist
+
+Whenever you are submitting an extension for approval, please check that you have completed the following tasks:
+
+* A release has been created in GitHub. You can create a release of the extension when syncing in the MakeCode editor or using the **pxt bump** cli command.
+* An icon image **icon.png** is available in the root folder. It should be 300x200 pixels and **less than 100kb** in size.
+* Descriptive GitHub description added to help with extension search.
```

Comment:
What's a "descriptive GitHub description"?

Reply:
description to help search